### PR TITLE
Add File-chooser demo

### PR DESCRIPTION
**Pull request by Linchenn**

### CODE REVIEW

File: examples/llm_inference/js/README.md
Comment (tyrmullen) [inline examples/llm_inference/js/README.md:1]: super-nit: I'd change the "Sample app 2" description just a little.  Maybe something like "This sample app has code which lends itself better to understanding the API."

Comment (Linchenn) [reply examples/llm_inference/js/README.md:1]: Done.

**Comment by PaulTR** [comment]
@schmidt-sebastian Looks like you're the awaiting reviewer.

**Comment by schmidt-sebastian** [comment]
We decided not to launch this for now due to added complexity. We wanted your input before proceeding.

**Pull request closed without merging**